sling/core/pckgmgr/src/main/java/com/composum/sling/core/pckgmgr/PackageServlet.java

```diff
@@ -139,7 +139,7 @@ public void init() throws ServletException {
         operations.setOperation(ServletOperationSet.Method.POST, Extension.json,
                 Operation.install, new InstallOperation());
         operations.setOperation(ServletOperationSet.Method.POST, Extension.json,
-                Operation.deploy, new DeployOperation());
+                Operation.deploy, new CrxServiceOperation());
```

Comment:
why do you use the name crx when its just apache sling and not aem :)? crx is the "custom" implementation of jcr of adobe as i understood it